web/README.md

```diff
@@ -15,8 +15,15 @@ This is a small example on how to use whisper-client on a web application.
     ```bash
     docker-compose up -d local
     ```
+
 3. Run the commands below;
 
     ```bash
     go build && ./whisper-example
     ```
```

Comment:
`docker-compose up` removes the necessity to have the right Golang SDK installed on the machine.  `docker-compose up --build` if the source code has changed.

Reply:
IMHO, it isn't necessary to upload a docker image of this code. The best option, seems to me, is to let it build always on usage. So the user would run only `docker-compose up`